x-pack/legacy/plugins/apm/public/utils/formatters.ts

```diff
@@ -5,6 +5,7 @@
  */
 
 import numeral from '@elastic/numeral';
```

Comment:
WDYT about splitting `formatters.ts` into more files? I'm seeing at least `duration_formatters.ts` and `datetime_formatters.ts`. Perhaps others too?

Reply:
Let's do it!!!